src/fsharp/service/ServiceAssemblyContent.fs

```diff
@@ -186,14 +187,31 @@ type IAssemblyContentCache =
 module AssemblyContentProvider =
     open System.IO
 
-    let private createEntity ns (parent: Parent) (entity: FSharpEntity) =
+    let unresolvedSymbol (topRequireQualifiedAccessParent: Idents option) (cleanedIdents: Idents) (fullName: string) =
+        let getNamespace (idents: Idents) = 
+            if idents.Length > 1 then Some idents.[..idents.Length - 2] else None
+
+        let ns = 
+            topRequireQualifiedAccessParent 
+            |> Option.bind getNamespace 
+            |> Option.orElseWith (fun () -> getNamespace cleanedIdents)
+            |> Option.defaultValue [||]
+
+        let displayName = cleanedIdents |> Array.skip ns.Length |> String.concat "."
```

Comment:
How do you know this `skip` isn't skipping past the end of the idents?  Perhaps be defensive?

Reply:
Done.